packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -1326,10 +1385,7 @@ describe('NetworkController', () => {
         {
           messenger,
         }: {
-          messenger: Messenger<
-            NetworkControllerActions,
-            NetworkControllerEvents
-          >;
+          messenger: RootMessenger;
```

Comment:
The type of the messenger in these tests only took internal actions and events into consideration and not external actions, so I've had to correct all of them.